testutil/cli/cmd.go

```diff
@@ -0,0 +1,45 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	cli2 "github.com/tendermint/tendermint/libs/cli"
```

Comment:
I would also prefer `tmcli` here
```suggestion
	tmcli "github.com/tendermint/tendermint/libs/cli"
```